packages/cxl-lumo-styles/scss/themes/vaadin-accordion-panel.scss

```diff
@@ -149,3 +149,42 @@
     padding-bottom: var(--lumo-space-m);
   }
 }
+
+/**
+ * Theme "cxl-dashboard-category".
+ */
+:host([theme~="cxl-dashboard-category"]) {
+  border-bottom: solid 1px var(--cxl-color-light-gray);
+
+  [part="summary"] {
+    padding: var(--lumo-space-l) 0;
+  }
+
+  [part="content"] {
+    padding: 0;
+    margin-bottom: var(--lumo-space-l);
+  }
+
+  [part="toggle"] {
+    margin-right: var(--lumo-space-l);
+  }
+}
+
+:host([theme~="cxl-dashboard-category"][opened]) {
+  [part="summary"] {
+    padding-bottom: 0;
+  }
+}
+
+@media (hover: hover) {
+  :host([theme~="cxl-dashboard-category"]:not([opened])) {
+    [part="summary"]:hover {
```

Comment:
I see `@media (hover: hover) {}` media query for the first time. It looks like an alternative for `:hover` but I'm confused why `:hover` needs to be repeated inside the MQ?

Reply:
` @media (hover: hover) {}` checks if the device supports hover and then the hover will be added if it's supported. So for example without it in mobile when someone clicks on the accordion there would be a quick background-color change which I didn't want to happen.